pygen/pygen_src/riscv_asm_program_gen.py

```diff
@@ -77,11 +80,17 @@ def gen_program(self):
 
             # Generate main program
             gt_lbl_str = pkg_ins.get_label("main", hart)
+            label_name = gt_lbl_str
             gt_lbl_str = riscv_instr_sequence()
             self.main_program.append(gt_lbl_str)
             self.main_program[hart].instr_cnt = cfg.main_program_instr_cnt
             self.main_program[hart].is_debug_program = 0
-            self.main_program[hart].label_name = "main"
+            self.main_program[hart].label_name = label_name
+            self.generate_directed_instr_stream(hart = hart,
```

Comment:
Minor formatting comment: According to pep8, having spaces around "=" in a keyword argument or a default parameter value is not recommended (this is for better code readability): https://www.python.org/dev/peps/pep-0008/#other-recommendations

Reply:
Got it @hodjat91 
Thanks!